raw_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "4.1.0-rc.7+amd64": {
+        "io.openshift.upgrades.graph.previous.add": "4.1.0-rc.6"
+    },
+    "4.1.0-rc.6+amd64": {
+        "io.openshift.upgrades.graph.previous.add": "4.1.0-rc.5"
+    },
+    "4.1.0-rc.8+amd64": {
+        "io.openshift.upgrades.graph.previous.add": "4.1.0-rc.7"
```

Comment:
This is redundant, because the 4.1.0-rc.7 -> 4.1.0-rc.8 edge is already compiled into the 4.1.0-rc.8 release.

```console
$ oc adm release info quay.io/openshift-release-dev/ocp-release:4.1.0-rc.8 | grep Upgrades
  Upgrades: 4.1.0-rc.7
```

We can drop the label from Quay without affecting the graph.

Reply:
@jwforres [removed this one too][1].

[1]: https://github.com/wking/cincinnati-graph-data/commit/ddc929278ec36bb2b2ad434db99994e7f292fbdb